src/main/java/com/example/SOPSbackend/model/Vaccine.java

```diff
@@ -0,0 +1,25 @@
+package com.example.SOPSbackend.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Vaccine {
```

Comment:
Należy zmienić nazwy encji na kończonce się na Entity, zrobiłem to w innym PR, tutaj proponuję zostawić.

IMO